addons/account/wizard/account_move_send.py

```diff
@@ -587,6 +587,8 @@ def action_send_and_print(self, from_cron=False, allow_fallback_pdf=False):
         else:
             for move in self.move_ids:
                 moves_data[move]['_form'] = self.new(self._get_available_field_values_in_multi(move))
+            # Marks moves as being sent asynchronously.
+            self.move_ids.is_being_sent = True
```

Comment:
Instead of an explicit flag, what about making a query looking to `account_move_account_move_send` table?

Reply:
:thinking: will try that tomorrow, I have no clue yet what's in that table, but good occasion start digging

Reply:
@clbr-odoo I don't know if it's exactly the name of the table but there is a cross-table created for the many2many move_ids